docs/NOW-spec.md

```diff
@@ -729,7 +700,10 @@ The NOW_VARU16 structure is used to encode unsigned integer values in the range
 | NOW_SYSTEM_INFO_RSP_ID<br>0x02 | NOW_SYSTEM_INFO_RSP_MSG |
 | NOW_SYSTEM_SHUTDOWN_ID<br>0x03 | NOW_SYSTEM_SHUTDOWN_MSG |
 
+<!-- TODO: Define NOW_SYSTEM_INFO_REQ_MSG, NOW_SYSTEM_INFO_RSP_MSG   -->
```

Comment:
Structures are not defined yet -- should we remove message kinds for them or they are to be defined?

Reply:
Maybe a leftover from the original document :thinking: